apps/nft-service/src/controllers/nft.controller.ts

```diff
@@ -16,7 +16,7 @@ export class NftController extends Controller {
    * it's associated claim code, and their address, and we transfer the token to them
    **/
   @Post('claimNft')
-  async claimNft(@Body() claim: NftClaim): Promise<boolean> {
+  async claimNft(@Body() claim: NftClaim): Promise<boolean|string> {
```

Comment:
boolean or string is this right?

Reply:
yeah.  More accurately, it could be `false|string`.  It could also be an empty string, but I thought it better to be explicit.